lib/dmv.rb

```diff
@@ -1,5 +1,9 @@
-class Dmv
+require './lib/facility'
+require './lib/dmv'
 
+class Dmv
+attr_reader :facilities
+attr_accessor :facilities
```

Comment:
You only need one of these. Both of these lines will create a `facilities` method that returns `@facilities`. But the attr_accessor will allow other classes to change the `@facilities` attribute. Looking through your code, I don't think you need to do that, so an attr_reader should be just fine.